### PR TITLE
[Core] Enhance constraints

### PR DESCRIPTION
**Pull request by loumalouomega**
This PR adds missing methods in order to enhance the constraints

### CODE REVIEW

File: kratos/includes/linear_master_slave_constraint.h [hunk 1]
```diff
@@ -330,18 +330,34 @@ class LinearMasterSlaveConstraint
...
-     * @param rSlaveDofList The list of slave DOFs
-     * @param rMasterDofList The list of slave DOFs
+     * @param rSlaveDofsVector The list of slave DOFs
+     * @param rMasterDofsVector The list of slave DOFs
      * @param rCurrentProcessInfo The current process info instance
      */
     void GetDofList(
-        DofPointerVectorType& rSlaveDofsVector,
-        DofPointerVectorType& rMasterDofsVector,
+        DofPointerVectorType& rSlaveDofList,
```
Comment (adityaghantasala) [inline kratos/includes/linear_master_slave_constraint.h]:
```suggestion
        DofPointerVectorType& rSlaveDofsVector,
```

File: kratos/includes/linear_master_slave_constraint.h [hunk 2]
```diff
@@ -330,18 +330,34 @@ class LinearMasterSlaveConstraint
...
+        DofPointerVectorType& rMasterDofList,
```
Comment (adityaghantasala) [inline kratos/includes/linear_master_slave_constraint.h]:
```suggestion
        DofPointerVectorType& rMasterDofsVector,
```

File: kratos/includes/linear_master_slave_constraint.h (hunk 1 above)
Comment (loumalouomega) [reply kratos/includes/linear_master_slave_constraint.h]: OK

**Comment by adityaghantasala** [comment]
So intend to create empty constraint then on add the master, slaves and their relations !. 
Fine for me !

**Comment by loumalouomega** [comment]
@adityaghantasala done

**Review by adityaghantasala — APPROVED** [review]

**Pull request merged**